docs/sources/storage/_index.md

```diff
@@ -42,6 +42,9 @@ S3 is AWS's hosted object store. It is a good candidate for a managed object sto
 Blob Storage is Microsoft Azure's hosted object store. It is a good candidate for a managed object store, especially when you're already running on Azure, and is production safe.
 You can authenticate Blob Storage access by using a storage account name and key or by using a Service Principal.
 
+### IBM Cloud Object Storage (COS)
```

Comment:
maybe a link to IBM Cloud Object Storage landing page?

Reply:
We felt is wiered to add link in the header, so we added cos landing page link in description.